lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -35,35 +35,19 @@ namespace Ohmbrewer {
             virtual const char* getType() const { return TemperatureSensor::TYPE_NAME; };
 
             /**
-             * The last temperature read by the sensor. Currently returns in Celsius.
-             * @returns A pointer to the Temperature object representing the last temperature reading
-             */
-            Temperature* getTemp() const;
-
-            /**
-             * The last time the temperature was read by the sensor
-             * @returns The last temperature reading time
-             */
-            int getLastReadTime() const;
-
-            /**
-             * Sets the last time the temperature was read by the sensor
-             * @param lastReadTime The last temperature reading time
-             */
-            const int setLastReadTime(const int lastReadTime);
-
-            /**
-             * @returns the probe for this sensor
+             * Constructor
+             * @param id The Sprout ID to use for this TemperatureSensor
+             * @param pins The list of physical pins this TemperatureSensor is attached to
              */
-            Probe* getProbe() const;
-
+            TemperatureSensor(int id, int busPin);
 
             /**
              * Constructor
              * @param id The Sprout ID to use for this TemperatureSensor
              * @param pins The list of physical pins this TemperatureSensor is attached to
+             * @param uid - the unique id code for the Dallas sensor connected to the bus.
              */
-            TemperatureSensor(int id, int busPin);
+            TemperatureSensor(int id, int busPin, char* uid);
```

Comment:
If you know how long the UID is (8, right?), you can limit the size as we've done in other parameters.